pgml-admin/app/views/projects.py

```diff
@@ -60,3 +67,208 @@ def get_context_data(self, **kwargs):
 class ProjectViewSet(viewsets.ModelViewSet):
     queryset = Project.objects.all()
     serializer_class = ProjectSerializer
+    filterset_fields = [
+        "name",
+    ]
+
+    @action(detail=False, permission_classes=[], methods=["POST"])
+    def train(self, request):
+        """Train a new project."""
+        serializer = NewProjectSerializer(data=request.data)
+        if serializer.is_valid():
+            with connection.cursor() as cursor:
+                cursor.execute(
+                    """
+                    SELECT * FROM pgml.train_joint(
```

Comment:
I'd use `train` here, unless you're letting them add multiple targets in the UX? I didn't see multiselect at that step. I'm not crazy about the current train/train_joint API split. It only exists to mirror `predict_joint`, because we can't have different return types (REAL vs REAL[]) on `predict()` and `predict_joint()`.

Reply:
Yup multi-select is allowed in the UI.